sdk/storage/storage-blob/src/Pipeline.ts

```diff
@@ -200,7 +180,7 @@ export function newPipeline(
   const factories: RequestPolicyFactory[] = [
     tracingPolicy(),
     new KeepAlivePolicyFactory(pipelineOptions.keepAliveOptions),
-    new TelemetryPolicyFactory(pipelineOptions.telemetry),
+    new TelemetryPolicyFactory(pipelineOptions.userAgentOptions),
```

Comment:
should we use the user agent policy factory from core-http? I guess that can be an additive change later